website/source/docs/agent/cloud-auto-join.html.md

```diff
@@ -102,6 +102,12 @@ $ consul agent -retry-join "provider=azure tag_name=... tag_value=... tenant_id=
 - `client_id` (required) - the client to authenticate with.
 - `secret_access_key` (required) - the secret client key. **NOTE** This value often may have an equals sign in it's value, especially if generated from the Azure Portal, so is important to wrap in single quotes eg. `secret_acccess_key='fpOfcHQJAQBczjAxiVpeyLmX1M0M0KPBST+GU2GvEN4='`
 
+   Variables can also be provided by environmental variables:
```

Comment:
This doesn't render quite right: 
![image](https://user-images.githubusercontent.com/17609145/54051155-1c379e00-41af-11e9-954e-8319cc0216b9.png)